daemon/graphdriver/overlay/overlay.go

```diff
@@ -99,6 +99,17 @@ func Init(home string, options []string) (graphdriver.Driver, error) {
 		return nil, graphdriver.ErrNotSupported
 	}
 
+	// check if they are running over btrfs
+	var buf syscall.Statfs_t
+	if err := syscall.Statfs(path.Dir(home), &buf); err != nil {
+		return nil, err
+	}
+
+	if graphdriver.FsMagic(buf.Type) == graphdriver.FsMagicBtrfs {
```

Comment:
@SvenDowideit  mentioned below request:
can we also add the underlying FS to docker info? (for all graph drivers that have them..)

it also need consider.